tests/unit_tests/model/test_model_parameter.py

```diff
@@ -87,6 +87,20 @@ def test_get_parameter_value(telescope_model_lst):
 def test_get_parameter_value_with_unit(telescope_model_lst):
     tel_model = telescope_model_lst
 
+    # check handling of list of values and units including null units
+    t_1 = tel_model.get_parameter_value_with_unit("focus_offset")
+    assert isinstance(t_1, list)
+    assert isinstance(t_1[0], u.Quantity)  # list of quantities returned
```

Comment:
Do you want to test in this case as well that unit  'null' in `t_1[2]` is converted correctly?

Reply:
Good idea, done.